docs/docs/05-reusable-components.md

```diff
@@ -239,6 +239,23 @@ Counter.propTypes = { initialCount: React.PropTypes.number };
 Counter.defaultProps = { initialCount: 0 };
 ```
 
+Alternatively, you can define `propTypes` and `defaultProps` as static properties of the class.
```

Comment:
Thanks for the feedback. I'll keep an eye on the PR and be happy to update after guidance.